Core1Platform/01_General.md

```diff
@@ -0,0 +1,17 @@
+# General information
+
+The *Actindo Core1 Platform* offers a comprehensive solution for efficient digital Unified Commerce. It is an API-first and cloud-native *Digital Operations Platform (DOP)* that provides you with an easy and fast integration of new business models in an existing IT landscape.   
+The platform brings together a high degree of automation and process integration. It offers a fast time to market as well as tremendous relief and time savings in day-to-day operations.   
+From product data, purchasing, and inventory to customer management, monitoring and reporting, everything is made available centrally in one place. 
+
+The *Actindo Core1 Platform* supports your business processes, regardless of the field of application you need a solution for:
+- You have an existing legacy system in use and want to switch from B2B (business-to-consumer) to D2C (direct-to-consumer).
```

Comment:
B2B = Business to business? 
D2C habe ich noch nicht gehört (was nicht heißt, dass es nicht richtig ist). Ich kenne es eher als B2C, als Gegensatz zu B2B. Meinst du das?